frontend/src/stories/components/RecommendationRequest/RecommendationRequestForm.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import RecommendationRequestForm from "main/components/RecommendationRequest/RecommendationRequestForm"
+import { recommendationRequestFixtures } from 'fixtures/recommendationRequestFixtures';
+
+export default {
+    title: 'components/RecommendationRequests/RecommendationRequestForm',
+    component: RecommendationRequestForm
+};
+
+
+const Template = (args) => {
+    return (
+        <RecommendationRequestForm {...args} />
+    )
+};
+
+export const Create = Template.bind({});
+
+Create.args = {
+    buttonLabel: "Create",
+    submitAction: (data) => {
+        console.log("Submit was clicked with data: ", data); 
+        window.alert("Submit was clicked with data: " + JSON.stringify(data));
+   }
+};
+
+export const Update = Template.bind({});
+
+Update.args = {
+    initialContents: recommendationRequestFixtures.oneRequest,
```

Comment:
Make sure that oneRequest is the right name. I'm pretty sure this should be oneRecommendation